example/gen/haberdasher_twirp.rb

```diff
@@ -1,7 +1,9 @@
 # Code generated by protoc-gen-twirp_ruby, DO NOT EDIT.
 require 'twirp'
 
-class HaberdasherService < Twirp::Service
-  PATH_PREFIX = "/twirp/examples.Haberdasher"
-  rpc :HelloWorld, Examples::HelloWorldRequest, Examples::HelloWorldResponse
+module Example
+  class HaberdasherService < Twirp::Service
+    PATH_PREFIX = "/twirp/example.Haberdasher"
+    rpc :HelloWorld, HelloWorldRequest, HelloWorldResponse, handler_method: :hello_world
+  end
```

Comment:
Changed in master: https://github.com/cyrusaf/ruby-twirp/commit/80abc62587d11656c43965b3e6cca701db15279b

The new DSL to build the Service should look like this:
```ruby
module Example
  class HaberdasherService < Twirp::Service
    package "example"
    service "Haberdasher"

    rpc "HelloWorld", HelloWorldRequest, HelloWorldResponse, handler_method: :hello_world
  end
end
```